agent/03_agent_deployment.sh

```diff
@@ -134,10 +199,34 @@ function attach_fleeting_iso() {
 
 write_pull_secret
 
-node0_ip=$(nth_ip $EXTERNAL_SUBNET_V4 20)
+# needed for assisted-service to run nmstatectl
+# This is temporary and will go away when https://github.com/nmstate/nmstate is used
+sudo yum install -y nmstate
+
+if [[ "$FLEETING_STATIC_IP_NODE0_ONLY" = "true" ]]; then
```

Comment:
Is this config variable required as a transient one?

Reply:
We need this now as the support for static ips on all states hasn't merged yet but I think it could also be used in the future as a case with static IP on node0 and dhcp addresses on all other hosts is a valid use case.